diff -rupN uClibc-ng-1.0.30/libc/sysdeps/linux/m68k/clone.S uClibc-ng-1.0.30-new/libc/sysdeps/linux/m68k/clone.S
--- uClibc-ng-1.0.30/libc/sysdeps/linux/m68k/clone.S	2018-04-28 15:16:33.000000000 -0400
+++ uClibc-ng-1.0.30-new/libc/sysdeps/linux/m68k/clone.S	2018-10-22 11:18:56.300533164 -0400
@@ -37,10 +37,12 @@ __clone:
 	/* Sanity check arguments.  */
 	movel	#-EINVAL, %d0
 	movel	4(%sp), %a0		/* no NULL function pointers */
-	tstl	%a0
+    movel   %a0, %d1
+	tstl	%d1
 	beq.w	__syscall_error_trampoline
 	movel	8(%sp), %a1		/* no NULL stack pointers */
-	tstl	%a1
+    movel   %a1, %d1
+	tstl	%d1
 	beq.w   __syscall_error_trampoline
 
 	/* Allocate space and copy the argument onto the new stack.  */
